snuba/clusters/cluster.py

```diff
@@ -241,6 +241,15 @@ def is_single_node(self) -> bool:
         """
         return self.__single_node
 
+    def get_cluster_name(self) -> Optional[str]:
+        return self.__cluster_name
+
+    def get_distributed_cluster_name(self) -> Optional[str]:
+        return self.__distributed_cluster_name
```

Comment:
Nobody is using this, could you please remove it. Also this feels like it should be an implementation detail of this class and we should try to avoid exposing it, otherwise everyone using an instance of this class would need to make sense of the difference between cluster_name and distributed_cluster_name.

Reply:
Done